core/server/master/src/test/java/alluxio/master/meta/InodeLockManagerTest.java

```diff
@@ -31,18 +31,18 @@
 public class InodeLockManagerTest {
   @Test(timeout = 10000)
   public void lockInode() throws Exception {
-    inodeLockTest(LockMode.WRITE, LockMode.READ, true);
-    inodeLockTest(LockMode.READ, LockMode.WRITE, true);
-    inodeLockTest(LockMode.WRITE, LockMode.WRITE, true);
-    inodeLockTest(LockMode.READ, LockMode.READ, false);
+    inodeLockTest(LockResource.LockMode.WRITE, LockResource.LockMode.READ, true);
+    inodeLockTest(LockResource.LockMode.READ, LockResource.LockMode.WRITE, true);
+    inodeLockTest(LockResource.LockMode.WRITE, LockResource.LockMode.WRITE, true);
+    inodeLockTest(LockResource.LockMode.READ, LockResource.LockMode.READ, false);
   }
 
   @Test(timeout = 10000)
   public void lockEdge() throws Exception {
-    edgeLockTest(LockMode.WRITE, LockMode.READ, true);
-    edgeLockTest(LockMode.READ, LockMode.WRITE, true);
-    edgeLockTest(LockMode.WRITE, LockMode.WRITE, true);
-    edgeLockTest(LockMode.READ, LockMode.READ, false);
+    edgeLockTest(LockResource.LockMode.WRITE, LockResource.LockMode.READ, true);
+    edgeLockTest(LockResource.LockMode.READ, LockResource.LockMode.WRITE, true);
+    edgeLockTest(LockResource.LockMode.WRITE, LockResource.LockMode.WRITE, true);
+    edgeLockTest(LockResource.LockMode.READ, LockResource.LockMode.READ, false);
```

Comment:
can we revert back to using `LockMode.X` instead of `LockResource.LockMode.X`?

Reply:
done